app/src/pages/registrationPage/registrationPage.tsx

```diff
@@ -12,23 +12,100 @@ export interface IRegistrationPageProps {
 export const RegistrationPage = (props: IRegistrationPageProps): React.ReactElement => {
   const history = useHistory();
   const [firstName, setFirstName] = React.useState<string | null>(null);
+  const [firstNameError, setFirstNameError] = React.useState<string | null>(null);
   const [lastName, setLastName] = React.useState<string | null>(null);
+  const [lastNameError, setLastNameError] = React.useState<string | null>(null);
   const [email, setEmail] = React.useState<string | null>(null);
+  const [emailError, setEmailError] = React.useState<string | null>(null);
   const [password, setPassword] = React.useState<string | null>(null);
-  const [retypedPassword, setRetypedPassword] = React.useState<string | null>(null);
+  const [passwordError, setPasswordError] = React.useState<string | null>(null);
+  const [confirmPassword, setConfirmPassword] = React.useState<string | null>(null);
```

Comment:
don't particularly like this name, but not too bad. any suggestions?

Reply:
maybe `confirmedPassword`? tough one lol